tests/integration/helpers/cluster.py

```diff
@@ -67,6 +67,17 @@ def get_odbc_bridge_path():
             return '/usr/bin/clickhouse-odbc-bridge'
     return path
 
+def get_docker_compose_path():
+    compose_path = os.environ.get('DOCKER_COMPOSE_DIR')
+    if compose_path is not None:
+        return os.path.dirname(compose_path)
+    else:
+        if os.path.exists(os.path.dirname('/compose/')):
+            return os.path.dirname('/compose/') #default in docker runner container
+        else:
+            print("Fallback docker_compose_path to LOCAL_DOCKER_COMPOSE_DIR: {}".format(LOCAL_DOCKER_COMPOSE_DIR))
```

Comment:
Almost everybody who runs it locally will see this message and it looks like you are making something wrong. Maybe something like `"Will use compose files from {}"`?

Reply:
Do you run integration tests without runner and runner docker container?
It is used ONLY when there is no `yandex/clickhouse-integration-tests-runner`. It means you run pytest manually without runner script.
And `yes` - it is a warning as it is not the same way CI runs it and it is better to see which YaMLs are used instead.

Reply:
`> Do you run integration tests without runner and runner docker container?`
To be honest, I always do)